### ISSUE TITLE
logits were turned off

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-pickling#175]
🤦‍♂️ noticed it after generating all embeddings for gpt2-xl

https://github.com/hassonlab/247-pickling/blob/dafb372f30669af5c70b4a758240f35b3ff9e1d8/scripts/tfsemb_genemb_causal.py#L262
